uwb: add WiMedia LLC Protocol (build system)

From: Greg Kroah-Hartman <gregkh@suse.de>

Add the WLP build system (Kconfig and Kbuild files).

Signed-off-by: David Vrabel <david.vrabel@csr.com>
---
 drivers/uwb/Kconfig      |    7 +++++++
 drivers/uwb/Makefile     |    1 +
 drivers/uwb/wlp/Makefile |   10 ++++++++++
 3 files changed, 18 insertions(+)

--- a/drivers/uwb/Kconfig
+++ b/drivers/uwb/Kconfig
@@ -57,4 +57,11 @@ config UWB_WHCI
           To compile this driver select Y (built in) or M (module). It
           is safe to select any even if you do not have the hardware.
 
+config UWB_WLP
+	tristate "Support WiMedia Link Protocol (Ethernet/IP over UWB)"
+	depends on UWB && NET
+	help
+	  This is a common library for drivers that implement
+	  networking over UWB.
+
 endif # UWB
--- a/drivers/uwb/Makefile
+++ b/drivers/uwb/Makefile
@@ -1,4 +1,5 @@
 obj-$(CONFIG_UWB)		+= uwb.o
+obj-$(CONFIG_UWB_WLP)		+= wlp/
 obj-$(CONFIG_UWB_WHCI)		+= umc.o whci.o whc-rc.o
 obj-$(CONFIG_UWB_HWA)		+= hwa-rc.o
 
--- /dev/null
+++ b/drivers/uwb/wlp/Makefile
@@ -0,0 +1,10 @@
+obj-$(CONFIG_UWB_WLP) := wlp.o
+
+wlp-objs :=	\
+	driver.o	\
+	eda.o		\
+	messages.o	\
+	sysfs.o		\
+	txrx.o		\
+	wlp-lc.o	\
+	wss-lc.o
